src/components/authorization/policies/by-role/financial-analyst.policy.ts

```diff
@@ -59,7 +59,7 @@ import {
             'financialReportPeriod',
             'financialReportReceivedAt',
           )
-          .when(member).edit,
+          .read.when(member).edit,
```

Comment:
Edit implies read, correct?  But in this case we want no member restrictions on reads so that's why the change?

Reply:
Exactly. We call out these properties because we want to _add_ **edit** access in certain cases. But because they are called out, the object level **read** is ignored.
So we need to "restore" that before widening the scope.